bril-rs/brillvm/runtime/src/main.rs

```diff
@@ -72,6 +72,11 @@ pub unsafe extern "C" fn _bril_parse_float(arg: *const c_char) -> f64 {
     r_str.parse::<f64>().unwrap()
 }
 
+#[no_mangle]
+pub extern "C" fn _start() -> ! {
```

Comment:
On Linux if I compile a test bril program with this version of the runtime with `clang-18`  I get a `/usr/bin/ld: duplicate definition of _start` error. 

Commenting this out and rebuilding the runtime has made it go away. I'm unsure if I'm missing some sort of linker option when I compile with clang, which would just be a documentation error.

Any way I can help with this? I can document/provide patch etc.

Reply:
So I took a couple of roundabouts while addressing your issue and I thought I still needed this for compilation. Seems we don't. Thanks for bringing this up!

Reply:
Unrelated but if you have any wisdom for getting an llvm installation in a github worker(especially for llvm 18) for testing that would be very useful.

Reply:
Hmmmmm, I have lots of github actions experience, a decent amount of LLVM experience, but no experience together.

The two projects I can point you to of examples of CI for LLVM for projects I have touched:

1. [Alive2](https://github.com/AliveToolkit/alive2/blob/master/.github/workflows/build.yml) works off of trunk, so it should be good to use with LLVM 18.
2. [TinyGo](https://github.com/tinygo-org/tinygo/blob/release/.github/workflows/linux.yml) Lots more complicated and they work off of a fork of LLVM, but lots of caching as well so they aren't having to set up LLVM all the time

If you want to mirror anyone's I'd recommend mirroring Alive's